invenio_cli/helpers/docker_helper.py

```diff
@@ -100,15 +114,16 @@ def destroy_containers(self):
 
     def execute_cli_command(self, project_shortname, command):
         """Execute an invenio CLI command in the API container."""
-        container_name = "{}_web-ui_1".format(self.container_prefix)
-        container = self.docker_client.containers.get(container_name)
-        status = container.exec_run(
-            cmd='/bin/bash -c "{}"'.format(command.replace('"', '\\"')),
-            tty=True,
-            stdout=True,
-            stderr=True,
-        )
-        # FIXME: What happens when exec_run fails? handle exception.
-        return ProcessResponse(
-            output=status.output.decode("utf-8").strip(), status_code=status.exit_code
-        )
+        container = self._get_container_from_service("web-ui")
+        if container:
```

Comment:
moderate: should we add an `else` clause and "hack" a custom `ProcessResponse` with e.g. `status_code=1` and `output="Web UI container not found. Is it up and running?`. Otherwise it would silently fail if no container (Is it possible to reach this part of the code with the containers down?)

Reply:
if you are following the high level api then containers should always be up. I can add a response :)

Reply:
Fair, still better safe than raise on runtime 😅